sdk/servicebus/Azure.Messaging.ServiceBus/src/Primitives/ServiceBusModelFactory.cs

```diff
@@ -0,0 +1,46 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.ComponentModel;
+
+namespace Azure.Messaging.ServiceBus.Primitives
```

Comment:
Not sure that we want to expose this new namespace - maybe drop the Primitives?

Reply:
Yeah. That's my bad. I wanted it to be the same namespace as the rest of the models. Making a new class in this folder automatically adds Primitives to it.

Reply:
Fixed!